README.md

```diff
@@ -1,2 +1,53 @@
-# EmailService
-Email micro service
+## EmailService
+Email micro service.  
+It's a simple service just to send emails.
+
+## Prerequriments
+
+.Net Core 3.1
+
+## Configuration
+
+```appsettings.Secret.json``` must be placed to src/EmailService folder
```

Comment:
Secret should be used for development.
Use just appsettings.json, appsettings.Production.json or just json configuration options